index.js

```diff
@@ -53,6 +54,11 @@ program
 
     const configFilePath = path.resolve(process.cwd(), '.escheckrc')
 
+    if (filesArg && filesArg.length && options.files) {
+      logger.error('Cannot pass in both [files...] argument and --files flag at the same time!')
+      process.exit(1)
+    }
```

Comment:
I'd originally thought to have the argument override the flag, but then I could see someone misunderstanding that maybe they got concatenated and it seemed like erroring is the only safe option.